docs/serving/configuration/feature-flags.md

```diff
@@ -78,8 +78,6 @@ Beta stage
 GA stage
 : The feature is allowed by default.
 
-# Available Flags
```

Comment:
Maybe we should keep this heading but change it to an H2. The heading provides separation between the information about how flags are configured, and the flags you can use. 

The "Available flags" heading should be an H2 header and the flags such as "Multiple containers" under it should be H3 headings.

Reply:
made the suggested change